scripts/circle_helper.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+function mapbox_time_start {
+    :
+}
+
+function mapbox_time_finish {
+    :
+}
+
+function mapbox_time {
+    :
+}
+
+function mapbox_fold {
+    :
+}
+
+function mapbox_nanoseconds {
+    :
+}
+
+export JOBS=${CIRCLE_NODE_TOTAL:-6}
```

Comment:
`CIRCLE_NODE_TOTAL` refers to the concurrently running nodes per build, not to thread concurrency.